libs/no-stream/src/index.tsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
 import { Heading, Text, VStack } from '@chakra-ui/react';
+import * as React from 'react';
 
-export const NoStream = () => {
+const NoStream = () => {
```

Comment:
**curiosity:** why did remove the named export?

Reply:
I was following the style of some of the components in the libs folder that instead did the default export at the bottom. I wasn't sure if there was a particular reason, but I was trying to match the style.